robot/Cumulus/resources/NPSP.py

```diff
@@ -1697,6 +1697,14 @@ def verify_table_contains_row(self,table_name,record,**kwargs):
                 print(f'Table contains {record} with empty {key} as expected')
             else:
                 raise Exception(f'Table did not contain {record} with expected {key}={value}')
-
-
-
+            
+    def get_org_name(self):
+        """
+        Obtains the orgname info from the cumulsci's get org info and returns the value
+        """
+        org_info = self.cumulusci.get_org_info()
+        if "sfdx_alias" in org_info.keys():
```

Comment:
The cumulusci library has an attribute named `org` which has a `name` attribute. You can simply return `self.cumulusci.org.name` instead of getting the org info and parsing the result.